modules/lang-groovy/src/test/java/org/elasticsearch/script/groovy/GroovyScriptTests.java

```diff
@@ -85,8 +85,7 @@ public void testGroovyExceptionSerialization() throws Exception {
         try {
             client().prepareSearch("test")
                     .setQuery(
-                            constantScoreQuery(scriptQuery(new Script("1 == not_found", ScriptType.INLINE, GroovyScriptEngineService.NAME,
-                                    null)))).get();
+                            constantScoreQuery(scriptQuery(new Script("1 == not_found", ScriptType.INLINE, "groovy", null)))).get();
```

Comment:
I think this one can still use the constant right? Maybe I'm missing which above comment you're referring to